test/test_read.ml

```diff
@@ -12,7 +12,57 @@ let from_file () =
   Alcotest.(check Testable.yojson) __LOC__ Fixtures.json_value (Yojson.Safe.from_file input_file);
   Sys.remove input_file
 
+let unquoted_from_string () =
+  Alcotest.(check Testable.yojson)
+    __LOC__
+    Fixtures.unquoted_value
+    (Yojson.Safe.from_string Fixtures.unquoted_json)
+
+let unquoted_from_lexbuf () =
+  let lexbuf = Lexing.from_string {|{foo: null, bar: "hello"}|} in
```

Comment:
Why not the more concise:
```suggestion
  let lexbuf = Lexing.from_string {|{foo: "hello"}|} in
```

Reply:
Because my idea to test `map_string` came as I was writing the test, but you're right, there's no point in making it this complicated and test unrelated things :+1: